compiler/rustc_metadata/src/rmeta/decoder.rs

```diff
@@ -700,28 +700,26 @@ impl MetadataBlob {
     }
 
     pub(crate) fn get_rustc_version(&self) -> String {
-        LazyValue::<String>::from_position(NonZeroUsize::new(METADATA_HEADER.len() + 4).unwrap())
+        LazyValue::<String>::from_position(NonZeroUsize::new(METADATA_HEADER.len() + 8).unwrap())
```

Comment:
I think this means we probably should bump the metadata version, since we can't read the rustc version reliably anymore?

Reply:
Hunh, that's exactly what the comment says. I really wish this system weren't implemented across so many different parts of the code :anguished: